sandstone/filewatcher.py

```diff
@@ -0,0 +1,58 @@
+from watchdog.observers import Observer
+from sandstone.lib.broadcast.manager import BroadcastManager
+from sandstone.lib.broadcast.message import BroadcastMessage
+import watchdog
+
+class FilesystemEventHandler(watchdog.events.FileSystemEventHandler):
+    """
+    Subclass of `watchdog.events.FilesystemEventHandler`
+    Manages on_created, on_deleted, on_moved events
+    """
+    def on_created(self, event):
+        """
+        Event Handler when a file is created
+        """
+        key = 'filetree:created_new_file'
```

Comment:
A minor thing, but renaming `created_new_file` to `created_file` would be more consistent.